app/views/users/_user.html.erb

```diff
@@ -0,0 +1,13 @@
+<article class="media is-align-items-center">
+  <div class="media-left">
+    <figure class="image is-128x128">
+    <%= image_tag user.photo %>
+    </figure>
+  </div>
+  <div class="media-content notification">
+    <div class="content">
+      <h4 class="title is-4"><%= user.name %></h4>
+      <p class="has-text-right is-size-7"><%= t('Posts_Counter', count: user.posts_counter)%></p>
```

Comment:
- [ ] The number of posts a user has isn't displaying on the view but rather being set as an HTML attribute. Kindly fix that.

View | HTML inspector
--|--
![image](https://user-images.githubusercontent.com/29071984/197717622-a7906f04-00b9-4bc9-b242-b380bd3d8655.png) | ![image](https://user-images.githubusercontent.com/29071984/197718089-65aebd9c-afab-40fc-8f57-470b856c2f8f.png)